src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    body,
+    title,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <UserInfo user={user} />
+
+        <p className="PostInfo__body">{body}</p>
+        <hr />
+        {comments.length > 0 ? (
```

Comment:
use a short evaluation
```suggestion
        {comments.length ? (
```